src/django/oar/urls.py

```diff
@@ -134,4 +134,8 @@
     path('api/geocoder/', views.get_geocoding, name='get_geocoding'),
 ] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
 
-urlpatterns = public_apis + internal_apis + info_apis
+opensearch_apis = [
```

Comment:
While naming this variable, I suggest focusing on `v1` rather than on OpenSearch.